arrpy/linalg.py

```
@@ -0,0 +1,63 @@
+    a : arrpy
+        Input array
+    axes : tuple of ints, optional
+        Permutation of axes
+    
+    Returns
+    -------
+    arrpy
+        Transposed array
+    """
+    # TODO: Implement transpose
+    pass
```

Ruff (0.12.2)

No newline at end of file

Add trailing newline

(W292)